config/nextflow.config

```diff
@@ -35,6 +35,11 @@ process {
         container = "shub://Molmed/summary-report-development:checkqc"
         }
 }
+process {
+    withName:GetMetadata {
+        container = "shub://Molmed/summary-report-development:checkqc"
```

Comment:
I guess the checkqc environment is not strictly necessary here? It could just use a clean python box instead?

Reply:
xml2dict is not included in the python base, so I think this is the most convenient solution for now.